client/my-sites/email/email-management/gsuite-user-item/index.jsx

```diff
@@ -20,47 +23,91 @@ import './style.scss';
 function GSuiteUserItem( props ) {
 	const translate = useTranslate();
 	const [ dialogVisible, setDialogVisible ] = useState( false );
-	const onFixClickHandler = ( e ) => {
-		e.preventDefault();
+
+	const onFixClickHandler = ( event ) => {
+		event.preventDefault();
+
 		setDialogVisible( true );
 	};
+
 	const onCloseClickHandler = () => {
 		setDialogVisible( false );
 	};
 
-	const getLoginLink = () => {
-		const { email, domain } = props.user;
-		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https://admin.google.com/a/${ domain }`;
+	const renderBadge = () => {
+		if ( ! props.user.is_admin ) {
+			return;
+		}
+
+		return (
+			<Badge type="info">
+				{ translate( 'Admin', { context: 'Noun: A user role displayed in a badge' } ) }
+			</Badge>
+		);
 	};
 
-	const renderManage = () => {
+	const renderMailboxLink = () => {
+		const { isSubscriptionActive, onClick, user } = props;
+
+		if ( ! isSubscriptionActive ) {
+			return;
+		}
+
 		return (
 			<ExternalLink
 				icon
-				className="gsuite-user-item"
-				href={ getLoginLink() }
-				onClick={ props.onClick }
+				href={ getGmailUrl( user.email ) }
+				onClick={ onClick }
 				target="_blank"
 				rel="noopener noreferrer"
+				title={ translate( 'Go to Gmail to access your emails' ) }
 			>
-				{ translate( 'Manage', { context: 'Login to G Suite Manage' } ) }
+				{ translate( 'Mailbox', { context: 'Link pointing to Gmail' } ) }
 			</ExternalLink>
 		);
 	};
 
-	const renderFix = () => {
+	const renderManageLink = () => {
+		const { onClick, user } = props;
+
+		if ( ! user.is_admin || ! user.agreed_to_terms ) {
+			return;
+		}
+
+		return (
+			<ExternalLink
+				icon
+				href={ getGoogleAdminUrl( user.email ) }
+				onClick={ onClick }
+				target="_blank"
+				rel="noopener noreferrer"
+				title={ translate( 'Go to Google Admin to manage your G Suite account' ) }
+			>
+				{ translate( 'Manage', { context: 'Link pointing to Google Admin' } ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **4** times:
`translate( 'Manage', { context: 'link to page to manage sites'} )` *ES Score: 11*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/gsuite-finish-setup-notices)


:information_source: This **context** is really long. Are you sure you don't want to use a translator comment instead?